parts/k8s/cloud-init/masternodecustomdata.yml

```diff
@@ -426,6 +426,11 @@ MASTER_CONTAINER_ADDONS_PLACEHOLDER
   content: |
     #!/bin/bash
     set -x
+    if [[ ! -s /etc/environment ]]; then
+        # /etc/environment is empty, which will break subsequent sed commands"
```

Comment:
Let's use go templating comments to save cloud-init overhead :)

E.g.:

```
{{- /* /etc/environment is empty, which will break subsequent sed commands"
       Append a blank line... */}}
```

Reply:
Done!